rpc/documentation/api.md

```diff
@@ -1476,3 +1478,43 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+#### `StartTicketPurchase`
+
+The `StartTicketPurchase` method enables ticket purchases for stake mining in the wallet.
+
+**Request:** `StartTicketPurchaseRequest`
+- `passphrase`: The private passphrase to unlock the wallet.
+- `account_name`: The name of the account being queried.
+- `balance_to_maintain`: The amount of funds to leave in wallet after purchasing tickets.
+- `max_fee`: The maximum ticket fee amount per KB.
+- `max_price_relative`: The scaling factor for setting the maximum ticket price, multiplied by the average price.
+- `max_price_absolute`: The maximum absolute ticket price.
+- `ticket_address`: The address to send all ticket outputs.
```

Comment:
I've been referring to this as the "voting address" and would like to see that name used here too.